x-pack/plugins/enterprise_search/public/applications/app_search/components/library/library.tsx

```diff
@@ -228,6 +231,28 @@ export const Library: React.FC = () => {
           <Result {...props} actions={actions} shouldLinkToDetailPage />
           <EuiSpacer />
 
+          <EuiSpacer />
+          <EuiTitle size="s">
+            <h3>With a drag handle</h3>
+          </EuiTitle>
+          <EuiSpacer />
+          <EuiDragDropContext onDragEnd={() => {}}>
+            <EuiDroppable spacing="m" droppableId="DraggableResultsTest">
+              {[1, 2, 3].map((_, i) => (
```

Comment:
This library example isn't meant to contain real reordering logic, hence the placeholder function/array.